tests/unit/views/pacbio/PacbioRun.spec.js

```diff
@@ -201,4 +208,17 @@ describe('Run.vue', () => {
         })
     })
 
+    describe('#reset', () => {
+
+        beforeEach(() => {
+            pacbioRun.showAlert = jest.fn()
+            pacbioRun.newRun = jest.fn()
+        })
+
+        it('calls newRun', async () => {
+            pacbioRun.newRun.mockReturnValue([])
+            pacbioRun.resetRun()
+            expect(pacbioRun.newRun).toBeCalled()
+        })
```

Comment:
If your mocking showAlert, you might want to add an expect showAlert to be called with ..